### DISCUSSION TITLE
compatibility with ESP32-A1S-Audio Kit

### DISCUSSION THREAD

**Discussion by gabodiazjayme** [forkineye/ESPixelStick#542] [General]
Is the code compatible with the ESP32-A1S-Audio Kit platform?       
![image](https://user-images.githubusercontent.com/102343317/176551966-e66e80c6-aecf-4629-bb35-f75d98ec2d48.png)

**Comment by MartinMueller2003** [comment]
Without looking at a schematic, I would say yes you could create a platform definition that would work on that card. You would have a maximum of 5 data output ports. If you are asking if there is a prebuilt image that will run on that card and do good things, then the answer is no. That card is not currently in the predefined platform set.
